frameworks/g-cloud-9/questions/services/dataSanitisationHosting.yml

```diff
@@ -0,0 +1,17 @@
+id: dataSanitisation
```

Comment:
Is it necessary to have `dataSanitisationSoftware` and `dataSanitisationHosting` in separate files even though the questions are identical?

Or should the `followup` values be different in these files?  e.g. instead of `dataSanitisationType` in both files should it be `dataSanitisationTypeHosting` here and `dataSanitisationTypeSoftware` in the other file?